alerts/google-gke/uptime-checks-workload.v1.json

```diff
@@ -0,0 +1,37 @@
+{
+  "displayName": "${CLUSTER_NAME}/${WORKLOAD_NAME} GKE Load Balancer Check uptime failure"
+  "documentation": {},
+  "userLabels": {
+    "workload_name": "${WORKLOAD_NAME}",
+    "cluster": "${CLUSTER_NAME}",
+    "uptime_check_id": "${UPTIME_CHECK_ID}",
+  },
+  "conditions": [
+    {
+      "conditionThreshold": {
+        "aggregations": [
+          {
+            "alignmentPeriod": "1200s",
+            "perSeriesAligner": "ALIGN_NEXT_OLDER",
+            "crossSeriesReducer": "REDUCE_COUNT_FALSE",
+	    "groupByFields": [
+              "resource.label.*"
+            ],
+          }
+        ],
+        "comparison": "COMPARISON_GT",
+        "duration": "${UPTIME_DURATION}",
+        "filter": "metric.type=\"monitoring.googleapis.com/uptime_check/check_passed\" AND metric.label.check_id=\"${UPTIME_CHECK_ID}\" AND resource.type=\"k8s_service\"",
```

Comment:
Same thing for ingress... passing in the resource.type instead of hardcoding

Reply:
ack, passing in the resource type